adding_species_manuscript.tex

```diff
@@ -194,6 +194,7 @@ \section*{Introduction}
 affiliated project networks, such as Vertebrate Genomes
 \citep{Rhie2021}, 10,000 Plants \citep{Cheng2018} and others \citep{darwin2022sequence},
 are providing the backbone for enormous increases in the amount of population-level genomic data available for model and non-model species.
+%REVISION: TBD - add more details and citations to this sentence
```

Comment:
I think we could restructure just a bit to more directly address the reviewer's concern by starting the paragraph with a piece of the following sentence, eg. "Population genetics allows us to answer questions across scales from deep evolutionary time to ongoing ecological dynamics, and dramatic reductions in sequencing costs are enabling the generation of unprecedented amounts of genomic data for a huge variety of species \citep{Ellegren2014}. Ongoing efforts..." 
Then "These data are being used to answer questions about adaptive introgression (eg. Gower et al. 2021), inference of population history and demographic parameters (eg. Speidel et al. 2021, Zang et al. 2021), and questions in non-model organisms such as distinguishing adaptation from drift (eg. Leigh et al. 2021) and understanding the implications of deleterious variation in populations of conservation concern (eg. Robinson et al. 2022)."

(I just grabbed some papers that cited Adrion et al. 2020)

Reply:
Looks great! I'll make these changes. I especially like the idea of citing papers that make use of stdpopsim :-)